demo/js/components/victory-zoom-container-demo.js

```diff
@@ -123,7 +118,7 @@ export default class App extends React.Component {
 
   getZoomDomain() {
     return {
-      y: [random(0, 0.4, 0.1), random(0.6, 1, 0.1)]
```

Comment:
ended up removing this since https://lodash.com/docs/4.17.15#random says the third argument should be a boolean. not sure if the `0.1` was intentional or if it should have a boolean value.